Arm/Hand/wrist_joint.hpp

```diff
@@ -93,10 +93,19 @@ class WristJoint
     return int(roll_offset_angle_);
   }
 
-  Acceleration acceleration;
-
  private:
 
+  /// Checks if value is zero. If it's zero make it not zero
+  float ReturnChangedIfZero(float acceleration)
+  {
+    if (acceleration == 0)
+    {
+      acceleration = 0.001;
+    }
+    return acceleration;
```

Comment:
could 1 line it using ternary operator like so:
`return acceleration == 0 ? 0.001 : acceleration;`